app/book.rb

```diff
@@ -0,0 +1,20 @@
+require_relative './item'
+class Book < Item
+  attr_accessor :publisher, :cover_state
+
+  # rubocop:disable Metrics/ParameterLists
+  # rubocop:disable Style/OptionalBooleanParameter
+  def initialize(name, publish_date, publisher, cover_state, archived = false, id = Random.rand(1..10_000))
+    super(name, publish_date, archived, id)
+    @publisher = publisher
+    @cover_state = cover_state
+  end
+  # rubocop:enable Metrics/ParameterLists
+  # rubocop:enable Style/OptionalBooleanParameter
+
+  private
+
+  def can_be_archived?
+    @cover_state == 'bad' || super
+  end
```

Comment:
- [ ] This should override the method from the parent class first and then should return true if the parent's method returns true OR if cover_state equals to "bad". The `super `should be evaluated first in the  || operator